deployment/run_server

```diff
@@ -26,7 +26,7 @@ fi
 pkill Xvfb
 pkill node
 
-xvfb-run --auto-servernum --server-args '-screen 0 640x480x24' ./bin/orca.js serve --request-limit=1000 --safe-mode $PLOTLYJS_ARG $@ 1>/proc/1/fd/1 2>/proc/1/fd/2 &
+xvfb-run --auto-servernum --server-args '-screen 0 640x480x24' ./bin/orca.js serve --safe-mode $PLOTLYJS_ARG $@ 1>/dev/stdout 2>/dev/stderr &
```

Comment:
Is there a reason to change to `/dev/stdout` and `/dev/stderr`? This wrapper is being run via monit, so `stdout` and `stderr` of the monit process are not necessarily the right place for this output.